### PR TITLE
Added verify option to docs

### PR DESCRIPTION
**Pull request by NigelVanHattum**
Added Docs for verify option to s3 remotes: [Pull request](https://github.com/iterative/dvc/pull/5732)

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/remote/modify.md [hunk 1]
```diff
@@ -182,6 +182,14 @@ these parameters, you could use the following options.
   ```dvc
   $ dvc remote modify myremote use_ssl false
   ```
+  
+- `ssl_verify` - whether or not to verify SSL certificates. By default SSL certificates are verified. You can provide the following values:
+  False - do not validate SSL certificates. SSL will still be used (unless use_ssl is False), but SSL certificates will not be verified.
+  path/to/cert/bundle.pem - A filename of the CA cert bundle to use.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/modify.md:188]: Please wrap to 80 chars 🙏

Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md:188]: And let us know when #5732 is approved in order to do a final check here 🙂

Comment (NigelVanHattum) [reply content/docs/command-reference/remote/modify.md]: @jorgeorpinel [#5732](https://github.com/iterative/dvc/pull/5732) has been merged.

**Comment by shcheklein** [comment]
Thanks @NigelVanHattum ! I've merged the restyled version of this PR

**Comment by jorgeorpinel** [comment]
Thanks guys. I didn't actually review this closely since the core PR looked like an early WIP... But np I'll make any copy edits or questions in another PR if needed 👍

**Pull request closed without merging**